src/firefox/index.js

```diff
@@ -164,9 +168,13 @@ export async function run(
     throw error;
   });
 
-  log.info(
-    'Use --verbose or open Tools > Web Developer > Browser Console ' +
-    'to see logging');
+  if (!verbose && !devtools) {
+    log.info('Use --verbose or --devtools to see logging');
+  }
+  if (devtools) {
+    log.info('More info about WebExtension debugging:');
+    log.info('https://extensionworkshop.com/documentation/develop/debugging/');
```

Comment:
@ochameau I'm thinking that we may as well always log this couple of messages, instead of condition them on the verbose and devtools flags.

wdyt?

As a side-effect, that may also make codecov to don't complain about these logs emitted on `devtools === true` not being covered in tests (which are not a big deal on their own, given that its just two informative logs I wouldn't definitely block on that codecov coverage failure).

Reply:
TBH, I'm not working enough with this tooling to have a strong opinion.

So I would prefer to defer to you/your team judgement.
I'm fine removing, or adding proper coverage.

Reply:
ok, let's remove the `verbose` flag propagation and keep that log unconditioned, while condition the new one to the `devtools` flag.

Let's also change `WebExtension` to `WebExtensions` in the new logged message and let's add something like this to `test.firefox.js` to explicitly cover it and make codecov happy again:
```
import {
  consoleStream, // instance is imported to inspect logged messages
} from '../../../src/util/logger.js';

...

    it('logs link to debugging docs', async () => {
      const runner = createFakeFxRunner();
      consoleStream.flushCapturedLogs();
      consoleStream.startCapturing();

      const expectedMessage = 'More info about WebExtensions debugging:';
      const expectedURLToDocs =
        'https://extensionworkshop.com/documentation/develop/debugging/';
      await runFirefox({fxRunner: runner, devtools: false});
      assert.notOk(consoleStream.capturedMessages.find(
        (msg) => msg.includes(expectedMessage)
      ));
      assert.notOk(consoleStream.capturedMessages.find(
        (msg) => msg.includes(expectedURLToDocs)
      ));

      consoleStream.flushCapturedLogs();

      await runFirefox({fxRunner: runner, devtools: true});
      const foundMessage = consoleStream.capturedMessages.find(
        (msg) => msg.includes(expectedMessage)
      );
      const foundDocURL = consoleStream.capturedMessages.find(
        (msg) => msg.includes(expectedURLToDocs)
      );

      // Expect the logs to be found.
      assert.ok(foundMessage);
      assert.ok(foundDocURL);

      // Expected to be emitted as info level logs.
      assert.ok(foundMessage?.includes('[info]'));
      assert.ok(foundDocURL?.includes('[info]'));
    });
```